manifests/o/Oracle/VirtualBox/6.1.40/Oracle.VirtualBox.installer.yaml

```diff
@@ -0,0 +1,40 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: Oracle.VirtualBox
+PackageVersion: 6.1.40
+Platform:
+- Windows.Desktop
+MinimumOSVersion: 10.0.0.0
+Scope: machine
+InstallModes:
+- interactive
+- silent
+InstallerSwitches:
+  Silent: --silent
+  SilentWithProgress: --silent
+  InstallLocation: --msiparams "INSTALLDIR="<INSTALLPATH>""
+  Custom: -msiparams REBOOT=ReallySuppress
+InstallerSuccessCodes:
+- 3010
+UpgradeBehavior: install
+FileExtensions:
+- ova
+- ovf
+- vdi
+- vhd
+- vmdk
+AppsAndFeaturesEntries:
+- DisplayName: Oracle VM VirtualBox 6.1.38
+  Publisher: Oracle Corporation
+  ProductCode: '{62A30AE6-8596-4C61-A5B8-1C3B45C25C72}'

```

Comment:
```suggestion
  ProductCode: '{43A0F3F1-1A26-43F3-ABD6-30E8A54D407E}'
```

Reply:
how is this determined? to avoid this issue in the future?

Reply:
> how is this determined? to avoid this issue in the future?

As the installer is an exe, you would have to install it to get the ProductCode unfortunately. It could be removed, and would install just fine, but previous submitters/maintainers added it.